package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Takuya Asano <takuya.a@gmail.com>",
   "description": "JavaScript implementation of Japanese morphological analyzer",
   "main": "./dist/node/kuromoji.js",
+  "browser": {
+    "./dist/node/loader/NodeDictionaryLoader.js": "./dist/node/loader/BrowserDictionaryLoader.js"
```

Comment:
`browser` field for external.(This affect external `browserify` command)

**external** is a user that use `kuromoji` as a module.

 `b.require` for internal.(This affect only `gulp build`)